docs/tutorial/building-a-theme.md

```diff
@@ -0,0 +1,1502 @@
+      </Main>
+    </ThemeLayout>
+  )
+}
+// highlight-end
+
+export default Layout
+```
+
+Run the site to see the theme changes that are starting to take effect:
+
+```shell
+yarn workspace site develop
+```
+
+![Theme UI changes starting to take effect on the site. For example, the header is now purple.](./images/building-a-theme-theme-ui-changes.png)
+
+To continue applying theme styles, you can use the `Style` import from Theme UI. For example, in the `event-list.js` component, change the `<h1>`, `<ul>` and `<li>` elements to reference their themed styles:
+
+```javascript:title=gatsby-theme-events/src/components/event-list.js
+import React from "react"
+import { Link } from "gatsby"
+// highlight-next-line
+import { Styled } from "theme-ui"
+
+const EventList = ({ events }) => {
+  return (
+    <>
+      // highlight-next-line
+      <Styled.h1>Upcoming Events</Styled.h1>
+      // highlight-next-line
+      <Styled.ul>
+        {events.map(event => (
+          // highlight-next-line
+          <Styled.li key={event.id}>
+            <strong>
+              <Link to={event.slug}>{event.name}</Link>
+            </strong>
+            <br />
+            {new Date(event.startDate).toLocaleDateString("en-US", {
+              month: "long",
+              day: "numeric",
+              year: "numeric",
+            })}{" "}
+            in {event.location}
+            // highlight-next-line
+          </Styled.li>
+        ))}
+        // highlight-next-line
+      </Styled.ul>
+    </>
+  )
+}
+
+export default EventList
+```
+
+By replacing the `h1` with `Styled.h1`, `ul` with `Styled.ul`, and `li` with `Styled.li`, the theme styles for those elements have been applied:
+
+![Theme UI style changes showing on the events listing.](./images/building-a-theme-events-listing-styling.png)
+
+## Publish a theme to npm
+
+By publishing your theme to npm, you make it available to pull in as a dependency for any of your projects, and for anyone in the community to use, too.
+
+### Namespace your theme
+
+It's important to namespace your theme. It helps differentiate between published packages, and avoid naming collisions.
+
+```json:title=gatsby-theme-events/package.json
+{
+  // highlight-next-line
+  "name": "yourname/gatsby-theme-events",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "gatsby build",
+    "clean": "gatsby clean",
+    "develop": "gatsby develop"
+  },
+  "peerDependencies": {
+    "gatsby": "^2.13.19",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
+  },
+  "devDependencies": {
+    "gatsby": "^2.13.19",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
+  },
+  "dependencies": {
+    "@emotion/core": "^10.0.14",
+    "@emotion/styled": "^10.0.14",
+    "@mdx-js/react": "^1.0.27",
+    "gatsby-plugin-theme-ui": "^0.2.6",
+    "gatsby-source-filesystem": "^2.1.5",
+    "gatsby-transformer-yaml": "^2.2.2",
+    "theme-ui": "^0.2.13"
+  }
+}
```

Comment:
One thing that I _don't_ cover in the egghead course that we should include here: the `keywords` need to be set or the theme won't show up in the showcase. 

These are the keywords required:
https://github.com/jlengstorf/gatsby-theme-jam-example/blob/7ac09a3c860aec02bcbbd7823133dc1925729df6/theme/package.json#L27-L31

Reply:
Do we want copies of the example themes in the course indexed, though?

Reply:
Hmmm... that's a good question. I was just talking about adding some kind of review process before themes show up in the showcase, so maybe we can kick this down the road a bit until we've decided how that will work?